src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1184,6 +1185,10 @@ void PolicyHandler::OnAllowSDLFunctionalityNotification(
             accessor.GetData().end(),
             DeactivateApplication(device_handle,
                                   application_manager_.state_controller()));
+#ifdef SDL_REMOTE_CONTROL
+        application_manager_.GetPluginManager().OnSDLEvent(
```

Comment:
@dev-gh suggest to add some function to AM like `NotifyEvent(event, app_id)` and with body like:
```
#ifdef SDL_REMOTE_CONTROL
plugin_manager_.OnSDLEvent(event, app_id);
#endif
```
Then you could add these calls in policy manager and other places without `ifdefs`
Also propose to move `SDLEvent` enum to AM namepace as it is more related to application manager, but not to plugins.

Reply:
@AKalinich-Luxoft 
I woudn't want to add one more interface to AM. I don't like much my approach either however we already have this GetPluginManager() interface. I was thinking of introducing another entity to handle that but it will require a lot of changes which maybe not accepted by customer.